xapian-letor/tests/Makefile.am

```diff
@@ -48,7 +49,7 @@ BUILT_SOURCES =
 
 if MAINTAINER_MODE
 BUILT_SOURCES += api_all.h api_collated.h $(collated_apitest_sources:.cc=.h) \
- api_generated.cc
+ api_generated.cc api_letor.cc
```

Comment:
Currently I have added it to collated_apitest_sources .... if that is wrong please let me know